manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       hostNetwork: true
       containers:
         - name: node-driver-registrar
-          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.1
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.6.0
```

Comment:
@jichenjc  , lets make it to latest, 2.6.2 , which creates the registration path . https://github.com/kubernetes-csi/node-driver-registrar/blob/release-2.6/CHANGELOG/CHANGELOG-2.6.2.md